public/app/features/admin/UpgradePage.tsx

```diff
@@ -125,8 +125,8 @@ const FeatureListing: React.FC = () => {
     <List>
       <Item title="Data source permissions" />
       <Item title="Reporting" />
-      <Item title="SAML Authentication" />
-      <Item title="Enhanced LDAP Integration" />
+      <Item title="SAML authentication" />
+      <Item title="Enhanced LDAP integration" />
       <Item title="Team Sync">LDAP, GitHub OAuth, Auth Proxy, Okta</Item>
       <Item title="White labeling" />
```

Comment:
Any chance this can be called "Rebranding"?

Reply:
Probably not, but that is a @mjseaman question.

Reply:
Yes good call out! I would like to call this feature "Custom branding." Open to debate on the name :) 

I think we should change it everywhere at once, and we'll have the opportunity to do that when we release some new functionality in an upcoming quarter. The new functionality would include custom UI colors and themes, and a user interface for uploading and managing custom graphics (as opposed to the .ini config options we have now). In the meantime I'd keep it as is.